pkg/kubelet/dockertools/manager.go

```diff
@@ -231,6 +237,7 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 		enableCustomMetrics:    enableCustomMetrics,
 		configureHairpinMode:   hairpinMode,
+		versionCache:           cache.NewVersionCache(),
```

Comment:
Oh, I noticed my mistake here, getVersionInfo() should also be invoked periodically. I will fix this by pass a updater to version cache just as your advice. Thanks!